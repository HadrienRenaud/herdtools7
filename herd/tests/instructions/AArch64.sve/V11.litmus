AArch64 V11
(* Contiguous store three-word structures from three vectors (scalar index) *)
{
 int x[12];

 0:X0=x;
}
P0                                       ;
 MOV Z1.S,#1                             ;
 MOV Z2.S,#2                             ;
 MOV Z3.S,#3                             ;
 MOV X1,#0                               ;
 PTRUE P0.S,VL4                          ;
 ST3W {Z1.S,Z2.S,Z3.S},P0,[X0,X1, LSL #2];

forall x={1,2,3,1,2,3,1,2,3,1,2,3}
