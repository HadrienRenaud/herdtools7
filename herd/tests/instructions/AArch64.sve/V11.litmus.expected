Test V11 Required
States 1
x={1,2,3,1,2,3,1,2,3,1,2,3};
Ok
Witnesses
Positive: 1 Negative: 0
Flag Scalable-Vector-Extensions-is-work-in-progress
Condition forall (x={1,2,3,1,2,3,1,2,3,1,2,3})
Observation V11 Always 1 0
Hash=a3c15b9b49c3645ba9113a9354c7a46a

